NDB.Covid19/NDB.Covid19.iOS/Utils/StressUtils.cs

```diff
@@ -30,7 +30,7 @@ public void Run(T1 v, T2 e)
 
             private void Reset()
             {
-                DispatchQueue.MainQueue.DispatchAfter(new DispatchTime(DispatchTime.Now, _delayMilliseconds * 10000000), () => { _hasStarted = false; });
+                DispatchQueue.MainQueue.DispatchAfter(new DispatchTime(DispatchTime.Now, _delayMilliseconds * 0), () => { _hasStarted = false; });
```

Comment:
Should the changes in this file be included?

Reply:
done